bson/datetime_ms.py

```diff
@@ -26,7 +26,7 @@
 from bson.tz_util import utc
 
 EPOCH_AWARE = datetime.datetime.fromtimestamp(0, utc)
-EPOCH_NAIVE = datetime.datetime.utcfromtimestamp(0)
+EPOCH_NAIVE = datetime.datetime.fromtimestamp(0, tz=datetime.timezone.utc).replace(tzinfo=None)
```

Comment:
Above we use `utc` from:
```python
from bson.tz_util import utc

EPOCH_AWARE = datetime.datetime.fromtimestamp(0, utc)
```

I'd suggest we use that instead of `datetime.timezone.utc` for consistency. We could open a new ticket to deprecate/remove tz_util since datetime.timezone has existed since Python 3.2: https://docs.python.org/3.6/library/datetime.html#datetime.timezone

Reply:
To clarify, you're suggesting we use the `utc` `bson.tz_util` specifically in the code above? Makes sense to me, although I agree we should deprecate tz_util to avoid duplicating the datetime.timezone functionality that has been present for years now.

Reply:
Yes I'm suggesting we use `bson.tz_util.utc` only on this line (and not elsewhere in the docs/tests).